drake/examples/Pendulum/CMakeLists.txt

```diff
@@ -12,14 +12,23 @@ if (LCM_FOUND)
   add_test(NAME runPendulumDynamics COMMAND runPendulumDynamics WORKING_DIRECTORY ${PROJECT_SOURCE_DIR}/examples/Pendulum)
 
   add_executable(runPendulumEnergyShaping runEnergyShaping.cpp)
-  target_link_libraries(runPendulumEnergyShaping drakeSystem)
+  target_link_libraries(runPendulumEnergyShaping drakeSystem drakeGeometryUtil drakeRBM)
   add_test(NAME runPendulumEnergyShaping COMMAND runPendulumEnergyShaping)
 
+  add_executable(runPendulumLQR runLQR.cpp)
+  target_link_libraries(runPendulumLQR drakeSystem drakeGeometryUtil drakeRBM)
+  add_test(NAME runPendulumLQR COMMAND runPendulumLQR WORKING_DIRECTORY ${PROJECT_SOURCE_DIR}/examples/Pendulum)
+
   write_procman(Pendulum.pmd
```

Comment:
cmake knows the absolute path to all of the files, so we can effectively install procman scripts for the client systems.